Sources/PortalSwift/Utils/Base58.swift

```diff
@@ -0,0 +1,122 @@
+//
+//  File.swift
+//
+//
+//  Created by Rami Shahatit on 5/7/24.
+//
+
+// Copyright Keefer Taylor, 2019.
+
+import BigInt
+import CommonCrypto
+import Foundation
+
+/// A static utility class which provides Base58 encoding and decoding functionality.
+public enum Base58 {
```

Comment:
We may want to call this `PortalBase58` in case customers have another Base58 library in their projects. For context, I've been trying to prepend all of our classes with `Portal` for this reason.